sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/QueueReceiverClient.cs

```diff
@@ -23,7 +23,7 @@ public class QueueReceiverClient : ServiceBusReceiverClient
         /// </summary>
         protected QueueReceiverClient(){}
         /// <summary>
-        ///   Initializes a new instance of the <see cref="ServiceBusReceiverClient"/> class.
+        ///   Initializes a new instance of the <see cref="ServiceBusReceiver"/> class.
```

Comment:
nit:  Doc comments refer to `ServiceBusReceiver` instead of `QueueReceiverClient`

Reply:
Fixed